pm/sendermonitor_test.go

```diff
@@ -470,49 +469,6 @@ func TestReserveAlloc(t *testing.T) {
 	assert.Zero(expectedAlloc.Cmp(alloc))
 }
 
-func TestValidateSender(t *testing.T) {
```

Comment:
This test is for `SenderMonitor.ValidateSender()` so we don't want to remove it.

Reply:
Oh yeah my bad, they looked the same at first glance, fixed acefc0d